src/main/java/com/google/devtools/build/lib/pkgcache/LoadingOptions.java

```diff
@@ -56,32 +66,32 @@
 
   @Option(
     name = "build_tag_filters",
```

Comment:
Both of these flags are documented in the Bazel User Manual (https://docs.bazel.build/versions/master/user-manual.html => bazel/site/docs/user-manual.html)

So, if you want to change the behavior of these flags, please also change the docs in the same commit. Thank you!

Reply:
Thanks for the comment! I pushed a new commit that updates the documentation of these options in bazel/site/docs/user-manual.html.

(@ehkloaj worked as a summer intern at our company, but has now gone back to college and is trying to leave his wild bazel days behind him. I will continue working with this pull request.)